dev-libs/boost/boost-1.82.0.ebuild

```diff
@@ -113,6 +108,15 @@ create_user-config.jam() {
 			-e 's/\[ unless \[ python\.numpy \] : <build>no \]/<build>no/g' \
 			-i "${BUILD_DIR}"/libs/python/build/Jamfile || die
 	fi
+
+	if [[ ${CHOST} == *-darwin* ]]; then
+		# On modern macOS, file I/O is already 64-bit by default,
+		# there's no support for special options like O_LARGEFILE.
+		# Thus, D_LARGEFILE64_SOURCE must be disabled.
+		sed \
+			-e 's/-D_LARGEFILE64_SOURCE//g' \
```

Comment:
instead of removing it here, can't you just be more selective when we pass the LFS flags?

Reply:
I do not understand how the Boost build system works. Do you have any suggestion on a better way to accomplish this?

Reply:
I now see, it's actually controlled by `append-lfs-flags`, I'll update it in a new revision.